dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldLayoutColumnSerializer.java

```diff
@@ -57,17 +54,24 @@ public void serialize(
 
         final List<Map<String, Object>> fieldsMap = jsonColumnsTransformer.mapList();
 
+        setFieldInternationalization(serializerProvider, fieldsMap);
+
+        jsonGenerator.writeObjectField("fields", fieldsMap);
+
+        jsonGenerator.writeEndObject();
+        jsonGenerator.flush();
+    }
+
+    private void setFieldInternationalization(SerializerProvider serializerProvider, List<Map<String, Object>> fieldsMap) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'serializerProvider' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)